frontend/js/src/explore/art-creator/components/IconTray.tsx

```diff
@@ -83,6 +86,29 @@ function IconTray(props: IconTrayProps) {
             </span>
           )}
         </div>
+        {browserHasClipboardAPI && (
+          <button
+            type="button"
+            className="btn btn-icon btn-info"
+            onClick={onClickCopyAlt}
+            style={{
+              marginLeft: "auto",
+              background: "transparent",
+              color: "darkgray",
+            }}
+          >
+            <text
+              style={{
+                textDecoration: "underline",
+                fontSize: "14px",
+                marginRight: "0.2em",
```

Comment:
Suggestion here:
 Somehow the `btn-xs` class will give us the margin we need (not ideal semantically, but it does the trick?) and as described above the `text-muted` class takes care of the color.
 
 Not sure we want the underline as it does not match any other button style anywhere on the website.
 
 
 
 Along with the other suggestions above the result is this:
![image](https://github.com/metabrainz/listenbrainz-server/assets/6179856/fe492e90-3a1d-4405-8cd4-a632940ab284)
It's not exactly the style you had before, so if you're unhappy with the result feel free to give this button its own CSS class and defined the CSS styles in the .less file.
Hope it makes sense !

Reply:
ended up just making a selector for that span, looks better this way imo